x-pack/plugins/reporting/server/core.ts

```diff
@@ -82,7 +83,7 @@ export interface ReportingInternalStart {
   licensing: LicensingPluginStart;
   logger: Logger;
   screenshotting?: ScreenshottingStart;
-  security?: SecurityPluginStart;
+  security?: SecurityServiceStart;
```

Comment:
1. This can be non-optional, since it is unconditionally provided by core.
2. Naming-wise, I lean toward having the field be called `securityService` to distinguish this from the deprecated APIs.
```suggestion
  securityService: SecurityServiceStart;
```